logger.js

```diff
@@ -208,44 +252,75 @@ logger._init();
 logger.log_if_level = function (level, key, plugin) {
     return function () {
         if (logger.loglevel < logger[key]) { return; }
-        var levelstr = '[' + level + ']';
-        var str = '';
-        var uuidstr = '[-]';
-        var pluginstr = '[' + (plugin || 'core') + ']';
+        var logobj = {
+            level,
+            connection_uuid: '-',
```

Comment:
Ok - it appears not.  This isn't going to be a connection_uuid, it could be a transaction ID or an outbound recipient ID.  So `uuid` is far more appropriate (and terse).